.storybook/main.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
   staticDirs: ['../app', './images'],
   env: (config) => ({
     ...config,
-    ENABLE_CONFIRMATION_REDESIGN: true,
```

Comment:
Since we support both redesigned and old screens should we not use the var in the stories that need it?

Reply:
The old screens and existing stories shouldn't be impacted by this since those components don't use this value, but this is a great convenience for the redesign stories so we don't have to repeatedly specify it.

Reply:
done